WooCommerce/UITestsFoundation/Screens/ExternalAppScreen.swift

```diff
@@ -9,7 +9,8 @@ public final class ExternalAppScreen {
     }
 
     let universalLinks = [
-        "payments": "https://woocommerce.com/mobile/payments"
+        "payments": "https://woocommerce.com/mobile/payments",
+        "orders": "https://www.woocommerce.com/mobile/orders/details?blog_id=161477129&order_id=3337"
```

Comment:
Super nitpicky nitpick (no change required). Leaving a comma at the line end will cause one line less noise for the next PR, if another link is added (as happened with L12):

```suggestion
        "orders": "https://www.woocommerce.com/mobile/orders/details?blog_id=161477129&order_id=3337",
```

Reply:
😄 i'll leave this as is for now, for now i think for now we're good with the two common universal links that are being used in emails so i think this list will not grow very soon